primitives/core/src/assertion/mod.rs

```diff
@@ -17,14 +17,33 @@
 // This file includes the predefined rulesets and the corresponding parameters
 // when requesting VCs.
 
-use crate::{
-	all_web3networks, AccountId, AchainableParams, BnbDigitDomainType, BoundedWeb3Network,
-	EVMTokenType, GenericDiscordRoleType, OneBlockCourseType, ParameterString, PlatformUserType,
-	VIP3MembershipCardLevel, Web3Network, Web3NftType, Web3TokenType,
-};
+pub mod achainable;
+pub mod bnb_domain;
+pub mod contest;
+pub mod evm_amount_holding;
+pub mod generic_discord_role;
+pub mod network;
+pub mod oneblock;
+pub mod platform_user;
+pub mod soraquiz;
+pub mod vip3;
+pub mod web3_nft;
+pub mod web3_token;
```

Comment:
Any reason to not follow the idiom
```
mod xxx;
pub use xxx::yyy;
```
?

Not trying to enforce it, maybe there's a case where this one fits better

Reply:
Not a particular reason, I'm just used to declare all modules at the top. But if this is convention we use I can change it to follow it